omnibus/.kitchen.yml

```diff
@@ -11,7 +11,9 @@ driver:
     cpus: 4
     memory: 4096
   synced_folders:
-    - ['../..', '/home/vagrant']
```

Comment:
@schisamo it mounts, but the `omnibus build chefdk` fails because it can't handle the root of `chefdk` being a mount link on Windows.